backend/src/models/user.rs

```diff
@@ -94,23 +109,95 @@ impl users::Model {
     }
 
     pub async fn create_with_password(
-        database: &DatabaseConnection,
-        uuid: SeaOrmUuid,
+        app_state: &web::Data<app_state::AppState>,
```

Comment:
This method is responsible for creating user with password. It is not responsible for auth flow. Revert it and move required stuff to auth service